app/src/app.config.json.tpl

```diff
@@ -11,7 +11,7 @@
     "iphoneUrl": "iosamw://",
     "androidUrlPart1": "intent:///",
     "androidUrlPart2": "#Intent;scheme=androidamw;package=com.alfresco.content.app;end",
-    "sessionTimeForOpenAppDialogDisplay": "${APP_CONFIG_SESSION_TIME_FOR_OPEN_APP_DIALOG_DISPLAY_IN_HOURS}",
+    "displayOpenAppDialogAfterHours": "${APP_CONFIG_SESSION_TIME_FOR_OPEN_APP_DIALOG_DISPLAY_IN_HOURS}",
```

Comment:
"after hours" means after the working day is over... you have key saying "after hours" but the value is "in hours", can you use something like "dialogTimeout" ?

Reply:
changed to "openInAppDialogTimeout"